Makefile

```diff
@@ -1,34 +1,43 @@
 # For people who don't want to use go's standard settings
 
-.PHONY: lit lit-af glit test tests
+.PHONY: lit lit-af glit test clean
 
 GOBIN = $(shell pwd)
 GO ?= latest
 WEBUI_REPO = "https://github.com/josephtchung/webui"
-all: lit test
 
+all: lit lit-af
 
-goget:
-	build/env.sh go get ./...
-
-lit: goget
-	build/env.sh go build
+lit:
+	build/env.sh go get -v .
+	build/env.sh go build -v
 	@echo "Done building."
 	@echo "Run \"$(GOBIN)/lit\" to launch lit."
 
-lit-af: goget
-	cd cmd/lit-af ; go build
+lit-af:
+	build/env.sh go get -v ./cmd/lit-af
```

Comment:
The reason `goget` is in place to remove this repeating `go get` thing, would be nice if we could add that back in.

Reply:
I tried to do that but it was complaining about glit still.  I can tweak it though to only download deps for things that actually get built.